### ISSUE TITLE
Developer Authentication

### ISSUE THREAD

**Issue by pg1671** [ozgur/python-linkedin#83]
Hi .. has developer authentication been removed by LinkedIn? I noticed today I had a number of failing calls with OAUTH errors and I went to check my developer keys and there is now no option to refresh them.  

Has this been removed or is it now only available for partners?

If so might be good to add a note to the page documentation here or remove the developer authentication notes and endpoint.

**Comment by NinshuBird** [comment]
I only have client ID and Client Secret ...

**Comment by thundergolfer** [comment]
@pg1671 yeah it looks like it has been removed. `linkedin.LinkedInAuthentication()` will be the way to go now.
